components/chef-run/lib/chef-run/ui/error_printer.rb

```diff
@@ -236,6 +236,7 @@ def add_formatted_backtrace(out)
         if backtrace_length > cause_trace.length
           out.print "\t... #{backtrace_length - cause_trace.length} more"
         end
+        out.print "\n"
```

Comment:
Found an issue where stack traces we print were squashed together in instances where there are nested errors